alvr/dashboard/src/steamvr_launcher/linux_steamvr.rs

```diff
@@ -150,9 +150,7 @@ fn linux_gpu_checks(device_infos: &[(&wgpu::Adapter, DeviceInfo)]) {
     let steamvr_root_dir = match alvr_server_io::steamvr_root_dir() {
         Ok(dir) => dir,
         Err(e) => {
-            error!("Couldn't detect openvr or steamvr files. \
-            Please make sure you have installed and ran SteamVR at least once. \
-            Or if you're using Flatpak Steam, make sure to use ALVR Dashboard from Flatpak ALVR. {e}");
```

Comment:
I'm really not happy with losing the flatpak steam information, because I feel that that's quite useful. We should find some way to still print it.

Reply:
Or actually, would two competing installations of flatpak steam steamvr and normal steamvr trip up steamlocate?
Since we need to make sure it'll find the directory of the one from flatpak steam, and this is actually realistic enough to occur on linux with users fucking around to see what actually works

Reply:
Gonna test specifically flatpak usecase, but steamlocate-rs does support finding even multiple installations, including flatpak (i thought it will probably find it relative to $home first, not from flatpak, but will check anyway)

Reply:
i realized that we might have a problem/solution for the flatpak problem

So in a nutshell, steamlocate-rs finds flatpak steam first, instead of native.
And we can launch either native or flatpak alvr (driver) from dashboard (which can be also launched from either flatpak or natively)
i wonder if we should allow user to choose from where to launch it?...

Reply:
i will make quick poc in this pr and see you opinion about this

Reply:
...not gonna make a quick poc, because RLS is not recognizing types for steamlocate::SteamDir::locate_multiple and even specifying them manually, doesn't make it work still... (compiler does work fine though)